spec/lib/hyrax/hirmeos/client_spec.rb

```diff
@@ -26,11 +26,14 @@
     end
   end
 
-  describe '#request_token' do
-    it 'makes a call to the token base url' do
-      token = client.request_token
-      expect(a_request(:post, "#{Hyrax::Hirmeos::MetricsTracker.token_base_url}/tokens")).to have_been_made.at_least_once
-      expect(token).to eq("exampleToken")
+  describe '#generate_token' do
+    it 'generates a token for authentication' do
+      sample_payload = {
+        "app": "hyku",
+        "purpose": "test"
+      }
+      token = client.generate_token(sample_payload)
+      expect(token).to eq('eyJhbGciOiJIUzI1NiJ9.eyJhcHAiOiJoeWt1IiwicHVycG9zZSI6InRlc3QifQ.tdZRtOkm9ZWPfig1_trrPLW8nKQsUQJXCS1IVYb4UZc')
```

Comment:
Instead of this token, how about describing the characteristics of `generate_token`'s return value like: 
```
expect(token).to be_present
```
and/or decode it and verify it has the expected payload hash.  I think this would work well with the default param value I suggested for `generate_token` then you could not pass a payload and test that the decoded token's payload matches what you expect (`iat` and `exp`).  There might also be some validation provided by the JWT gem that could be used here.